tests/ballerina-integration-test/src/test/java/org/ballerinalang/test/checkpointing/InterruptibleServiceTestCase.java

```diff
@@ -57,6 +59,10 @@ public void setup() {
             statesStorageDir.delete();
         }
         String statesStoragePath = statesStorageDir.getAbsolutePath();
+        String osName = Utils.getOSName();
+        if (osName != null && osName.toLowerCase(Locale.ENGLISH).contains("windows")) {
+            statesStoragePath = statesStoragePath.replace("\\", "\\\\");
```

Comment:
Can't we use logic like this: https://github.com/ballerina-platform/ballerina-lang/pull/9690/files#diff-6c9342002ed6ed8411beee56a5b7be0aR49

Reply:
@hasithaa Here file path is correctly computate. Problem here is when we pass file path to ballerina as parameter, we need to put \\ in windows environment. 

ex. ballerina run <balFile> -e C:\\user\\tempfileDir

AFAIU above commit logic does not solve that.